driverkit/Makefile

```diff
@@ -5,33 +5,53 @@ ifeq ($(DRIVERKIT),)
 DRIVERKIT := "/bin/driverkit"
 endif
 
-CONFIGS := $(wildcard config/*/*.yaml)
+# Recursive wildcard
+rwildcard=$(foreach d,$(wildcard $(1:=/*)),$(call rwildcard,$d,$2) $(filter $(subst *,%,$2),$d))
+
+CONFIGS := $(call rwildcard,config/*,*.yaml)
```

Comment:
Find all configs under config and its subfolders (eventually, for supported $arch).